content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -429,447 +429,6 @@ Some values of an object are typically generated before the object is persisted.
 * Any field set by a mutating admission controller
 * For the `Service` resource: Ports or IPs that kube-apiserver assigns to v1.Service objects
 
-## Server Side Apply
```

Comment:
Can we leave a breadcrumb so people at least have a link to the new page from here?

Reply:
Yes, I agree.

Reply:
Looks like the original SSA heading remains with the sentence on line 434.